build-tools/automation/build.groovy

```diff
@@ -188,9 +188,9 @@ timestamps {
         stageWithTimeout('publish test error logs to Azure', 10, 'MINUTES', '', false) {  // Typically takes less than a minute
             echo "packaging test error logs"
 
-            sh "make -C ${XADir} -k package-test-errors"
+            sh "make -C ${XADir} -k package-test-results"
 
-            def publishTestFilePaths = "${XADir}/xa-test-errors*,${XADir}/test-errors.zip"
+            def publishTestFilePaths = "${XADir}/xa-test-results*,${XADir}/test-errors.zip"
 
```

Comment:
I think these new names make sense, we will just need to also update things in monodroid:
https://github.com/xamarin/monodroid/blob/d32a22583ff44588876091fe8e9cb4c98ca39f44/Jenkinsfile#L360